physics.py

```diff
@@ -154,5 +155,9 @@ def update_sim(self, now: float, tm_diff: float) -> None:
             )
         )
 
-        self.vision_sim.adjustCamera(self.camera, self.visual_localiser.robot_to_camera)
-        self.vision_sim.update(self.physics_controller.get_pose())
+        # Simulate slow vision updates.
+        self.vision_sim_counter += 1
+        if self.vision_sim_counter == 10:
```

Comment:
This is probably slower than what we really get from PhotonVision on our Raspberry Pi 5. I do see the console warnings about our code being slow though, so I'm erring on the slower end for simulation.